dist/webcam-easy.js

```diff
@@ -53,14 +59,16 @@ class Webcam {
         } else {
             videoConstraints.deviceId = { exact: this._selectedDeviceId};
         }
+        /* Try to get a media stream that takes at least 1080p pictures */
+        videoConstraints.height = { min: 1080 }
```

Comment:
min means minimum supported resolution. So if you have smaller resolution camera you will not see it. I think here could be used `ideal `. 


> To require a capability, use the keywords min, max, or exact
https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia